azure-container-apps-arm/README.md

```diff
@@ -0,0 +1,103 @@
+
+## Before you begin
+
+Before you begin, complete the necessary [preparation steps to deploy 1Password SCIM Bridge](/PREPARATION.md). You'll also need an Azure account with permission to create a Container App.
+
+If you are using Google Workspace as your identity provider, follow [step 1](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client) to create a Google service account, key, and API client. The json key file you downloaded will be your `workspace-credentials` in step 2 of this documentation.
+
+> [!NOTE]
+> If you don't have an Azure account, you can sign up for a free trial with starting credit: https://azure.microsoft.com/free/
+
+## Step 1: Download the template file
+1. Download the [ARM(Azure Resource Manager) template](./aca-op-scim-bridge-template.json) file using the download icon at the top right.
+
+## Step 2: Create the Container App
+
+1. Sign in to the Azure Portal and go to the [Deploy a custom template](https://portal.azure.com/#create/Microsoft.Template) page.
+2. Click **Build your own template in the editor**.
+3. Click **Load file** and upload the template file you downloaded earlier. Then click **Save**.
+4. Fill out the following fields:
+   - **Subscription** : Choose the subscription you prefer.
+   - **Resource group**: Choose an existing Resource Group or create a new one using **Create new** button.
+   - **Region**: Choose the region you prefer.
+   - **Container App Name**: Enter a name you'd like to use, default will be `op-scim-con-app`.
+   - **Container App Env Name**: Enter a name you'd like to use, default will be `op-scim-con-app-env`.
+   - **Container App Log Analytics Name**: Enter a name you'd like to use, default will be `op-scim-con-app-env`.
+   - **Scimsession** : Paste the entire contents of the `scimession` file.
+   - **Workspace Actor** *(Only if you are using Google Workspace as your identity provider)*: Enter the email address of a Google Workspace administrator for the service account. If you are not using Google Workspace as your identity provider, leave this field blank.
+   - **Workspace Credentials** *(Only if you are using Google Workspace as your identity provider)*: Paste the entire contents of the json key file you downloaded in the [before you begin](#before-you-begin) section. If you are not using Google Workspace as your identity provider, leave this field blank.
+> [!IMPORTANT]
+> If you are not using Google Workspace as your identity provider, leave `Workspace Actor` and `Workspace Credential` fields blank.
+3. Click **Review + create**.
+4. Once the validation succeeds, click **Create**. It will take a couple of minutes to run the deployment.
+
+
+## Step 3: Test your SCIM bridge
+Once your deployment is complete, click **Go to resource group** and click on the Container App you created.
```

Comment:
Case (Container Apps is the product name, but MS uses lower case for each individual instance in their docs, so we should do the same):

```suggestion
Once your deployment is complete, click **Go to resource group** and click on the container app you created.
```